handbook.md

```diff
@@ -80,6 +80,16 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple, distinct issues you can follow this example:
+
+```
+Fixes #1234,#5678 - Content view yum publish bug
+
+Repositories were not being referenced correctly. 
```

Comment:
As previous comment (https://github.com/theforeman/theforeman.org/pull/732#discussion_r87147507), why include this paragraph? It's not relevant to the example.